zlib-rs/src/inflate/bitreader.rs

```diff
@@ -120,7 +120,7 @@ impl<'a> BitReader<'a> {
         // SAFETY: assertion above ensures we have 8 bytes to read for a u64.
         let read = unsafe { core::ptr::read_unaligned(self.ptr.cast::<u64>()) }.to_le();
         self.bit_buffer |= read << self.bits_used;
-        let increment = (63 - self.bits_used) >> 3;
+        let increment = (63 ^ self.bits_used) >> 3;
```

Comment:
@mat-1 could you add a comment here that we're really doing a subtraction here, but use `xor` for performance reasons?

Reply:
I apologize for taking so long to get back to you, this should now be resolved.